Packs/CommonWidgets/Scripts/MyToDoTasksWidget/README.md

```diff
@@ -0,0 +1,21 @@
+A script that creates a table of all the ToDo tasks assigned to the current user.
+
+## Script Data
+
+---
+
+| **Name** | **Description** |
+| --- | --- |
+| Script Type | python3 |
+| Tags | widget |
+| Cortex XSOAR Version | 6.1.0 |
+
+## Inputs
+
+---
+There are no inputs for this script.
+
+## Outputs
+
+---
+Markdown table of open tasks assigned to the current user.
```

Comment:
please run generate-outputs with the `-e` argument so that we have some run examples here (make sure they don't include any sensitive info before comitting)

Reply:
Done, added an example human readable output.